lte/gateway/c/oai/tasks/nas/emm/msg/emm_msg.c

```diff
@@ -172,6 +174,13 @@ int emm_msg_decode(EMM_msg* msg, uint8_t* buffer, uint32_t len) {
     case SECURITY_MODE_COMPLETE:
       decode_result = decode_security_mode_complete(
           &msg->security_mode_complete, buffer, len);
+      // MAX_IMEISV_SIZE is defined as 15, but IMEISV is
```

Comment:
Does it make sense to modify the MAX_IMEISV_SIZE to be 16? I see it is being used in the Sgs messages, which might need more testing.

Reply:
yes, let's confirm the implication on SGS if we change the size to 16. Then we can revisit all usages in a separate PR.